localstack-core/localstack/constants.py

```diff
@@ -1,10 +1,12 @@
 import os
 
-from localstack.version import __version__
+from localstack.version import __version_tuple__
 
-# temporary fix to avoid issues with missing version package in image
-# FIXME: remove this once all images contain the version package
-VERSION = __version__
+# Adapt the version from setuptools_scm to the version format used by LocalStack before
+if len(__version_tuple__) > 3:
+    VERSION = ".".join([str(i) for i in __version_tuple__[0:3]]) + ".dev"
+else:
+    VERSION = ".".join([str(i) for i in __version_tuple__[0:3]])
```

Comment:
I don't think this is necessary anymore (since we use `local_scheme = "no-local-version"`).